specification/_types/mapping/meta-fields.ts

```diff
@@ -61,4 +61,15 @@ export class SourceField {
   enabled?: boolean
   excludes?: string[]
   includes?: string[]
+  mode?: Mode
+}
+
+export enum Mode {
```

Comment:
You can remove the `= N` on enums. Also since this is a top-level class in `specification/_types` we should give it a specific name, maybe `SourceFieldMode`?

Reply:
Done!